core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/extensions.scala

```diff
@@ -138,3 +138,5 @@ case class RenameConstraint(oldName: String, newName: String) extends TableAlter
 case class RenameColumn(oldName: String, newName: String) extends TableAlteration
 
 case class AlterTableCommand(tableName: String, alterations: Seq[TableAlteration]) extends Catalog {}
+
+case class ExpressionList(expressions: Seq[Expression]) extends Expression {}
```

Comment:
why do we need this?

Reply:
It has gone. I changed the grammar but that is in another PR. I will rebase when that goes through